dbinit/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM yonadev/hsqldb:2.3.2
+
+ADD DatabaseInitializer.jar DatabaseInitializer.jar
+ADD data data
+
+RUN ["java","-jar","DatabaseInitializer.jar","--spring.datasource.url=jdbc:hsqldb:file:thedatabase"]
```

Comment:
Shouldn't the connection string be externalized? Because once MariaDB is in place this will change again.